web-api/src/storage/elastic.rs

```diff
@@ -444,7 +444,7 @@ struct DocumentPropertiesResponse {
 }
 
 #[derive(Clone, Debug, Deserialize)]
-struct IgnoredResponse;
+struct IgnoredResponse {}
```

Comment:
❓ from a rust type perspective this is the same, but if i get it right this is needed because of the json serde, right? then we should add a comment about this implicit detail, such that it doesn't get changed accidentally again.

Reply:
Unit  structs and empty structs are just mostly the same but differences in their handling is not limit to `serde` but also other derives and some other aspects like type matching/destruction and construction. 

But yes, it's for serde to deserialize it as an empty struct instead of an unit type.

I added documentation.